site/gatsby-site/page-creators/createCitiationPages.js

```diff
@@ -8,19 +44,94 @@ const createCitiationPages = (graphql, createPage) => {
           query IncidentIDs {
             allMongodbAiidprodIncidents {
               distinct(field: incident_id)
+              group(field: incident_id) {
+                fieldValue
+                edges {
+                  node {
+                    id
+                    submitters
+                    incident_date
+                    date_published
+                    incident_id
+                    report_number
+                    title
+                    url
+                    image_url
+                    source_domain
+                    mongodb_id
+                    text
+                    authors
+                  }
+                }
+              }
             }
 
-            allMongodbAiidprodDuplicates {
+            allMongodbAiidprodClassifications(
+              filter: { classifications: { Publish: { eq: true } } }
+            ) {
               nodes {
-                true_incident_number
-                duplicate_incident_number
+                incident_id
+                id
+                namespace
+                classifications {
```

Comment:
I think you previously collected the classifications queries elsewhere? I know we are likely to change the classifications from objects to an array, so I am concerned by spreading this out.

Also, how did this change land in the fix to the links in the citation pages?